src/routes/game/components/elements/cardPortal/CardPortal.test.tsx

```diff
@@ -0,0 +1,8 @@
+import CardPortal from './CardPortal';
```

Comment:
This cardPortal component is just a renamed component as this used to be CardPopUp. But it wasn't really the one doing the pop-ing up but was just the portal to show the card(s).